_posts/2019-09-23-good-first-issue.md

```diff
@@ -0,0 +1,140 @@
+| [alexellis](https://github.com/alexellis) | - [derek](https://github.com/alexellis/derek)<br/>- [github-exporter](https://github.com/alexellis/github-exporter)<br/>- [inlets](https://github.com/alexellis/inlets)<br/>- [minikube-operator](https://github.com/alexellis/minikube-operator)<br/>- [ubiquitous-octo-guacamole](https://github.com/alexellis/ubiquitous-octo-guacamole) |
+| [google](https://github.com/google) | [go-github](https://github.com/google/go-github) |
+| [reactiverse](https://github.com/reactiverse) | [es4x](https://github.com/reactiverse/es4x) |
+| [jetstack](https://github.com/jetstack) | [cert-manager](https://github.com/jetstack/cert-manager) |
+| [nut-tree](https://github.com/nut-tree) | - [nut.js](https://github.com/nut-tree/nut.js)<br/>- [secrets](https://github.com/nut-tree/secrets)<br/>- [trailmix](https://github.com/nut-tree/trailmix) |
+| [sakuli](https://github.com/sakuli) | [sakuli](https://github.com/sakuli/sakuli) |
+| [Ewocker](https://github.com/Ewocker) | [vue-lodash](https://github.com/Ewocker/vue-lodash) |
+
+### Limitations
+
+* At the time of this writing, `goodfirstissue` bot relies on the owner of project/org to enable webhooks for sending push notifications when issues are created/updated.
+* It does not tweet about onboarding of new projects. It will be nice, as a follower of this twitter account, to know which new projects will start sending `goodfirstissue` issues our way.
+
+### Future Enhancements
+
+* Ability to poll for events in a scalable way for projects that support `goodfristissue` but are reluctant to install too many webhook integrations on the project/org.
+* Ability to publish metrics around how many `goodfirstissue` were tweeted, how many contributors actually benefited from it. And we can do these metrics along a few dimensions like Programming language, UI/Backend/Infra etc. 
+
+### Wrapping up
+
+The goal of ```goodfirstissue``` bot is to connect first time contributors with great projects giving opportunity to learn and contribute and we recently crossed 100 followers on twitter account.
+
+* If you are maintainer of an open source project, we would highly appreciate you considering [installing](https://github.com/apps/goodfirstissue) this on your org/project.
+* If you are new to open source community and looking for a starting point, consider following [@goodfirstissue](https://twitter.com/goodfirstissue).
+* If you like this project, it will encourage us if add a star to the [repo](https://github.com/rajatjindal/goodfirstissue). (we are one star away from first magical number `10`).
+* You can also follow [Rajat Jindal](https://twitter.com/rajatjindal1983) who is your fellow "first time contributor".
+
+### Acknowledgements
+
+Many thanks to [@alexellisuk](https://twitter.com/alexellisuk/)] for helping me write, build and deploy this [@openfaas](https://twitter.com/openfaas/) function to openfaas-cloud.
```

Comment:
There is an extra `]` at the end of the `@alexellisuk` link

Reply:
thanks again. fixed it.